clamav/client/api.py

```diff
@@ -33,7 +33,7 @@ class ClamAVApi(object):
     @ensure_annotations
     def __init__(
         self,
-        routes:ClamAVRoutes,
+        routes: ClamAVRoutes,
```

Comment:
please separate this commit from this PR; merge it to master. cosmetic changes like that do not need to be reviewed. we have a linter for that